src/entity/user/UserComment.ts

```diff
@@ -34,8 +34,8 @@ export class UserComment {
   @Column({ default: null, nullable: true })
   votedAt: Date;
 
-  @Column({ type: 'smallint', default: UserVote.None })
-  vote: UserVote = UserVote.None;
+  @Column({ type: 'smallint', default: 0 })
+  vote: UserVote = 0;
```

Comment:
Why don't we use the enum here anymore? Was it an issue?

Reply:
Dependency cycle, I'll sort it.

Reply:
Should be solved with 255ce7051d825ff95cbe5cb685ea9aad5541335e

Reply:
```suggestion
  vote: UserVote = UserVote.None;
```